src/Concerns/RejectsRecords.php

```diff
@@ -12,34 +12,34 @@
 trait RejectsRecords
 {
     /**
-     * @var ?callable(CacheEvent): bool
+     * @var array<callable(CacheEvent): bool>
      */
-    private $rejectCacheEventCallback = null;
+    private $rejectCacheEventCallbacks = [];
```

Comment:
🍸 Do we wanna add a type here? We couldn't before as they were `callable`.


```suggestion
    private array $rejectCacheEventCallbacks = [];
```

Reply:
Yeah, we should! Updated in [a5e81e3](https://github.com/laravel/nightwatch/pull/287/commits/a5e81e32fe63df46d4b616322b9505c63577dced)